trainTestReinforcementAlgorithm.py

```diff
@@ -138,7 +185,13 @@ def main():
     aparser.add_argument("--train_timesteps", type=int, default=1000)
     
     aparser.add_argument("--test_timesteps", type=int, default=100)
+    aparser.add_argument("--board_height", type=int, default=10)
+    aparser.add_argument("--board_width", type=int, default=10)
+    aparser.add_argument("--visualize_training", type=bool, default=False)
     aparser.add_argument("--visualize_testing", type=bool, default=True)
+    aparser.add_argument("--visualization_fps", type=int, default=30)
+
+    aparser.add_argument("--reward_function", type=Callable[..., float], default=basic_reward_func, help="function to determine how a snake agent is rewarded/punished for certain actions during training. Available functions can be found in snakeRewardFuncs.py")
```

Comment:
Can basic_reward_func work as a command line argument? How would you pass this in?

Reply:
Maybe we should pass in the name of the reward function as a command line argument then parse that in `snake_env.py`?